PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/webapp/config/RiderWebAppRunState.kt

```diff
@@ -136,6 +134,12 @@ class RiderWebAppRunState(project: Project,
     override fun onSuccess(result: Pair<WebApp, SqlDatabase?>, processHandler: RunProcessHandler) {
         processHandler.notifyComplete()
 
+        AzureUIRefreshCore.addListener("Refresh after publish", object : AzureUIRefreshListener() {
+            override fun run() {
+
```

Comment:
I assume this is empty intentionally?

Reply:
Fixed. Thx. Just forgot to remove this pease of code.